CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ The `next` version of 4.29 is now available. Planned release date is February 20
 
 ## Breaking Changes
 
-- For local builds, [Webpack](https://webpack.js.org/) versions prior to `5.84.0` have been removed at 4.29. This is related to bug fixes in Webpack.
+- For local builds, using [Webpack](https://webpack.js.org/) versions prior to `5.84.0` is not recommended for 4.29. This is related to bug fixes in Webpack.
```

Comment:
Thanks @steveoh. I agree the sentence could use some adjustment and we've already tweaked it a few times, so I'm going to leave it for now. We'll revisit for the final release notes. It is odd because we mixed a gerund (using) with several nouns.